WcaOnRails/app/models/user.rb

```diff
@@ -471,9 +471,25 @@ def can_vote_in_poll?
     admin? || results_team? || any_kind_of_delegate? || wrc_team?
   end
 
+  def can_view_delegate_matters?
+    any_kind_of_delegate? || can_admin_results? || wrc_team? || wdc_team?
+  end
+
+  def can_manage_incidents?
+    admin? || wrc_team?
+  end
+
+  def can_view_incident_private_sections?(incident)
+    if incident.resolved_at
```

Comment:
nit: It might make some of this code a little nicer if you define a `resolved?` method.